src/bundle/Resources/public/js/scripts/core/dropdown.js

```diff
@@ -250,6 +260,30 @@
             }
         }
 
+        filterItems(event) {
+            const allItems = this.itemsListContainer.querySelectorAll('[data-filter-value]');
+
+            if (event.currentTarget.value.length < 3) {
+                [...allItems].forEach((item) => item.classList.remove(CLASS_ITEM_HIDDEN));
+
+                return;
+            }
+
+            const visibleItems = this.itemsListContainer.querySelectorAll(`[data-filter-value^="${event.currentTarget.value.toLowerCase()}"]`);
```

Comment:
Code may be easier to read after assigning `event.currentTarget.value.toLowerCase()` to some variable.